lib/Importer/Caffe2.cpp

```diff
@@ -270,9 +270,20 @@ void caffe2ModelLoader::loadOperator(const caffe2::OperatorDef &op) {
 
     Node *node = G_.createConcat(opName, inputs, channel);
 
+    unsigned_t addAxis = dict.count("add_axis") ? loadInt(dict["add_axis"]) : 0;
+
+    Node *concatOutput = node;
+
+    if (addAxis) {
+      // When add axis is used, this means we have to add a new dimension before
+      // the axis, instead of merging on the axis.
+      std::vector<size_t> outputDims = inputs[0].dims();
```

Comment:
Should we also check that dims for inputs[1], inputs[2], ... match?

Reply:
Good point!

Reply:
Done.

Added a couple of GLOW_ASSERTs for that.